package.json

```diff
@@ -19,9 +19,10 @@
     "url": "https://github.com/jsoverson/grunt-contrib-jasmine-example/issues"
   },
   "devDependencies": {
-    "grunt-contrib-jasmine": "~0.4.0",
+    "grunt-contrib-jasmine": "^1.0.3",
     "grunt": "~0.4.0",
-    "grunt-contrib-jshint": "~0.2.0"
+    "grunt-contrib-jshint": "~0.2.0",
+    "jasmine": "^2.5.3"
```

Comment:
Do you need to include this library? I've run the tests without including `"jasmine": "^2.5.3"` and it's all worked fine.